File: lib/app/models/application_entity/application_entity_test.exs
defmodule ApplicationEntityTest do
  use DataCase

  describe "master entity" do
    test "expect masster entity to exist" do
      assert DBTestUtils.get_count("application_entity") == 1

      assert DB.query_val("SELECT (pub_id) FROM application_entity WHERE type = 'MASTER'") ==
               "MASTER"
    end
  end
end

File: pkg/models/application_entity/application_entity_test.go
funcmodule ApplicationEntityTest {
  use DataCase

  describe "master entity" {
    test "expect masster entity to exist" {
      assert DBTestUtils.get_count("application_entity") == 1

      assert db.QueryVal("SELECT (pub_id) FROM application_entity WHERE type = 'MASTER'") ==
               "MASTER"
    }
  }
}
